meetings/2021-09-06.md

```diff
@@ -0,0 +1,72 @@
+# 2021-09-06 Authentication Panel
+
+https://meet.jit.si/solid-authentication
+
+## Agenda
+
+* Feedback from CSS implementers on Sep 13th (2 min)
+* Scheduling AS discussion in AuthZ panel (2 min)
+* Documenting delegation scenarios (2 min)
+* [Determine identifier and title for the solid-oidc spec #27](https://github.com/solid/solid-oidc/issues/27)
+* WebID (where do we expect End-user's and Client's (Web/D)ID Documents to be hosted?)
+  * [Define Solid-OIDC requirements for what client description WebID Document should include #28](https://github.com/solid/solid-oidc/issues/28)
+  * [Proposal: change webid claim to solid #26](https://github.com/solid/solid-oidc/issues/26)
+  * [Advertising trusted OIDC issuers via Link headers #32](https://github.com/solid/solid-oidc/issues/32)
+  * [Generalize Solid-OIDC protocol beyond WebID #35](https://github.com/solid/solid-oidc/issues/35)
+
+## Present
+
+* Nicolas
+* Pavlik
+* Barath
+
+
+## Minutes
+
+### Feedback from CSS implementers on Sep 13th
+
+Nicolas: We can also discuss client credentials flow on that meeting. I know that Ruben and Joachim were
+looking into implementing client credentials flow.
+Pavlik: I know that Justin also uses client credentials flow some places
+
+### Scheduling AS discussion in next AuthZ panel
+
+Pavlik: Given that Matthieu is on vacation we should do it next week Wed 15th
+
+### Documenting delegation scenarios
+
+Pavlik: I discussed some of such use cases with Aaron but we should discuss it again when he's present. 
+There are two active related scenarios in the AuthZ panel:
+- https://solid.github.io/authorization-panel/authorization-ucr/#uc-delegation-subset
+- https://solid.github.io/authorization-panel/authorization-ucr/#group-membership-vc
+
+
+### where do we expect End-user's and Client's (Web/D)ID Documents to be hosted?
+
+Pavlik: Currently NSS and CSS create WebID documents in solid storage.
+Nicolas: I think this question should be out of the question for solid-oidc spec. We want to avoid coupling between
+WebID document and storage. For exapmle if storage is provided by pod provider one can loose one's WebID.
+Ideally one would have WebID hosted somewhere which redirects to solid pod or elsewhere. In case of webapps,
```

Comment:
```suggestion

Nicolas: I think this should be out of the question for solid-oidc spec. We want to avoid coupling between
WebID document and storage. For example, if storage is provided by pod provider, one can loose one's WebID.
Ideally, one would have WebID hosted somewhere which redirects to solid pod or elsewhere. In case of webapps,
```